packages/fuels-accounts/src/provider.rs

```diff
@@ -253,7 +253,7 @@ impl Provider {
             .map_err(Into::into)
     }
 
-    #[cfg(feature = "fuel-core")]
```

Comment:
I'm out of context: why are we making this change here?

Reply:
Oh it's just that the feature is called `fuel-core-lib` and that this one is called `fuel-core`. Unless we happen to have a `fuel-core` feature that's different but in that case it's not documented.

Reply:
Btw this kind of proves a fear I had which is that we are pretty much not testing anything related to the `fuel-core-lib` feature.

Reply:
Hmm, that's true. I can't seem to find any feature called just `"fuel-core"`, shouldn't it fail if we conditionally compile something with a feature that doesn't exist? 🤔 In any case, lgtm.